iceoryx_examples/icediscovery/README.md

```diff
@@ -22,9 +22,9 @@ the availability of services respectively.
 We create several publishers which offer their services on construction by
 default. For more dynamism the `cameraPublishers` offer/stop their services
 periodically. If you want more information on how to create a publisher, have a
-look at the [icehello](https://github.com/eclipse-iceoryx/iceoryx/tree/v2.0.0/iceoryx_examples/icehello),
-[icedelivery](https://github.com/eclipse-iceoryx/iceoryx/tree/v2.0.0/iceoryx_examples/icedelivery),
-and [iceoptions](https://github.com/eclipse-iceoryx/iceoryx/tree/v2.0.0/iceoryx_examples/icedelivery)
+look at the [icehello](https://github.com/eclipse-iceoryx/iceoryx/tree/master/iceoryx_examples/icehello),
```

Comment:
Why do we link to master in the icediscovery examples?

Reply:
@FerdinandSpitzschnueffler After our discussion last week, we came up with the following proposal:

* `master` branch always points to `latest` on [iceoryx.io](https://iceoryx.io) and `master` for all GitHub links
* `release_x.0` uses specific version tags which are increased before new releases are published e.g. `vx.0.1` or `vx.1.0`, etc.

Reply:
Oh, I overlooked that the target branch is master...